dev-utils/storybook/stories/lit-components/button.stories.js

```diff
@@ -0,0 +1,102 @@
+import { html } from "lit-element";
+import "../../../../feature-utils/poly-look/src/lit-components/buttons";
+import { themeConfiguration } from "./themeConfiguration";
+
+export default {
+    title: "Visuals/Atoms/Button",
+    component: "poly-button",
+    argTypes: {
+        type: { control: "text" },
+        size: { control: "text" },
+        disabled: { control: "boolean" },
+        onPolyButtonClicked: {
+            action: "polyButtonClicked",
+        },
+    },
+};
+
+function Template({
```

Comment:
I'm seeing several invocations to Template, and first, they are very similar, so I wonder if they can be factored out to a base class; this would extend to whatever is returned, which, at the end of the day, is a string. However, there are differences; for instance, some use "theme", some others use "type".  "label" is used in some cases only. So two things here:
1. Is it possible to un-DRY, avoiding boilerplate, by factoring out common code to a base class? For instance, 
https://github.com/polypoly-eu/polyPod/pull/732/files#diff-9ec1ed48ed4669b5ce596a6b87b5fa27a597895fc13ed4c6bd3f738c38cddc46R27-R34
seems to appear several times.
2. Can we avoid constants like the one used for background-color? (which, besides, might clash with the theme)

Reply:
I would have to look into this. Everything that's under `lit-components` is what we already had, so I'm not familiar with that code.

In general, I'm not sure how much boilerplate is avoidable, but at this stage I'm more tempted to leave it as it is. The reason is that we are still learning how to use Storybook so keeping things simple might be more beneficial. Once we have more experience we should look into reducing boilerplate.